ClientApp/src/components/admin/overview/ConnectButton.tsx

```diff
@@ -1,18 +1,35 @@
-import { Button } from "@material-ui/core";
-import React from "react";
+import React, {useState} from "react";
 import useStyles from "./Styles";
 import { SelectedConnectionType } from "../../../common/components/Types";
+import { Grid, Typography, Modal, Box, Button} from "@material-ui/core";
+
 
 type ConnectButtonProps = {
   selectedConnection: SelectedConnectionType;
   connectGiverRecipient: () => void;
 };
 
+const style = {
```

Comment:
See comment before last commit :)

Reply:
Yes, that previous commit was not supposed to be pushed to this merge :)